tested/dsl/translate_parser.py

```diff
@@ -94,6 +92,24 @@ class ReturnOracle(dict):
 )
 
 
+def build_preprocessor_messages(
```

Comment:
I think this would better fit in `dsl_errors` as this is a way to handle a specific type of error in the `dsl`